.gitignore

```diff
@@ -0,0 +1,10 @@
+# Ignore log file
+.nextflow.log*
+.nextflow/
+.nextflow/history
+
+# Ignore generated folders
+results/
+work/
+data/
+bio-test-datasets/
```

Comment:
I added this to ensure we dont bload this repo with test data. Instructions on how to download have been added to readme.